keyboards/mokey/luckycat70/info.json

```diff
@@ -0,0 +1,195 @@
+{
+    "keyboard_name": "Lucky Cat 70",
+    "manufacturer": "qmk",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x6653",
+        "pid": "0x0070",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": false,
+        "backlight": false,
+        "rgblight": false,
+        "rgb_matrix": true,
+        "audio": false
+    },
+    "matrix_pins": {
+        "cols": ["A3", "A2", "A5", "A6", "A7", "B0", "B1", "B10", "B11", "B12", "B13", "B14", "B15", "A8", "A9", "A10", "A15"],
+        "rows": ["C13", "C14", "C15", "A0", "A1"]
+    },
+    "ws2812": {
+        "pin": "A4"
+    },
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "board": "STM32_F103_STM32DUINO",
+    "rgb_matrix": {
+        "driver": "WS2812",
+        "layout": [
+            {"flags": 1, "matrix": [0, 0], "x": 0, "y": 0},
```

Comment:
A flag of `1` configures as a "modifier". I very much doubt is every single key on the keyboard.